tests/dataframe/test_drop_pytest.py

```diff
@@ -15,17 +15,24 @@
 #  specific language governing permissions and limitations
 #  under the License.
 
+import pytest
+
 # File called _pytest for PyCharm compatability
+from tests.common import TestData
 
 
-class TestDataFrameDrop:
+class TestDataFrameDrop(TestData):
     def test_drop(self, df):
-        df.drop(["Carrier", "DestCityName"], axis=1)
+        df.drop(labels=["Carrier", "DestCityName"], axis=1)
```

Comment:
Can we leave at least one without `labels=`.

Reply:
Yes